docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding.md

```diff
@@ -0,0 +1,146 @@
+
+{% endinfo_block %}
+
+The official relationship between the operator and merchant can be established by providing a contract to a merchant. For specific merchants, you can use custom contracts. If not, boilerplate agreements should be readily available. Contracts can be created as part of the terms and conditions, which must be accepted beforehand, along with the platform rules, customer communication guidelines, data policy, and SLA agreement. Clarification is also required regarding who the product data owner is, in which categories the merchant is allowed to sell, and the commissions and fees the merchant must pay the operator.
+
+![edit-merchant](https://spryker.s3.eu-central-1.amazonaws.com/docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding/step-2-edit-merchant.png)
+
+### 3) Operator: Approve the merchant
+
+1. In the Back Office, navigate to **Marketplace&nbsp;<span aria-label="and then">></span> Merchants**.
+2. On the **Overview of Merchants** page that opens, in the **Actions** column, click **Approve Access**. 
+
+By [approving the merchant](/docs/pbc/all/merchant-management/{{site.version}}/marketplace/manage-in-the-back-office/manage-merchants.html#approving-and-denying-merchants), the operator enables merchant users to access the Merchant Portal to complete their public-facing profile and create products and offers. 
+Then, the operator needs to create a merchant admin user based on data provided through the registration form, such as the email or first and last name of the contact person. For this, the operator finds a merchant user in the Back Office, in **Merchant&nbsp;<span aria-label="and then">></span> Users** and assigns them the required permissions.
+The email with the password is automatically sent to access the Merchant Portal later on. The operator can also create more user accounts for the merchant if required.
+
+![overview-of-merchants](https://spryker.s3.eu-central-1.amazonaws.com/docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding/step-3-overview-of-merchants.png)
+
+![edit-user](https://spryker.s3.eu-central-1.amazonaws.com/docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding/step-3-edit-user.png)
+
+### 4) Merchant and operator: Create and update the product and offers
+
+Operator: Decide who is the owner of the product data:
+* If the operator is the owner and manages the product data, other merchants can create offers with specific prices and available quantities on top of existing products.
+* If merchants can create products, the operator needs to make sure there are no duplicates in the Marketplace and regularly check the product data quality.
+
+In the Merchant Portal, merchants can create and update products and offers individually. Alternatively, the operator can help the merchant set up the process for automatically importing products and offers from a CSV template or establish the data exchange process from the merchant's PIM or ERP system.
```

Comment:
With my comment to line 107, I actually meant that it would be good to add the link to the doc on how to create products in the merchant portal, specifically, to another new page you have created in this PR